ibis/backends/dask/aggcontext.py

```diff
@@ -0,0 +1,235 @@
+import operator
+from typing import Any, Callable, Dict, Tuple, Union
+
+import dask.dataframe as dd
+from dask.dataframe.groupby import SeriesGroupBy
+
+import ibis
+import ibis.expr.datatypes as dt
+from ibis.backends.pandas.aggcontext import (
+    AggregationContext,
+    compute_window_spec,
+    construct_time_context_aware_series,
+    get_time_col,
+    window_agg_udf,
+    wrap_for_agg,
+)
+
+
+class Summarize(AggregationContext):
+    __slots__ = ()
+
+    def agg(self, grouped_data, function, *args, **kwargs):
+        if isinstance(function, str):
+            return getattr(grouped_data, function)(*args, **kwargs)
+
+        if not callable(function):
+            raise TypeError(f'Object {function} is not callable or a string')
+
+        if isinstance(grouped_data, SeriesGroupBy) and len(grouped_data):
```

Comment:
Codecov seems to believe that none of these branches get hit with tests. Can you confirm whether that's the case?

Reply:
Yeah, this branch isn't getting hit. Dask aggregations hit slightly different code paths than Pandas. So the tests aren't exercising it in the same way.

Reply:
Do we need this code then? Or we will in the future?

Reply:
I've left it around because this is exactly what Pandas is doing right now.  
When we improve Dask test coverage it should hit this branch.